bump_stargate.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ -z ${1+x} ]; then 
+   echo "stargate version is a required argument"
+   exit 1
+fi
+
+STARGATE_VERSION=$1
+find . -name "Dockerfile" -exec perl -i -pe "s|ARG STARGATE_VERSION=.*|ARG STARGATE_VERSION=${STARGATE_VERSION}|" {} \;
+find . -name "start_*.sh" -exec perl -i -pe "s|export SGTAG=.*|export SGTAG=${STARGATE_VERSION}|" {} \;
```

Comment:
+1 to automating version bump, but to take it one step further, shouldn't we standardize on the Stargate version variable name too is this case?

Reply:
Let's file a follow up issue?

Reply:
Not even sure what @dimas-b is pointing to. Let's have it in another PR/ticket.